Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -410,15 +410,15 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
  * @parameter range attributed string range to use.
  * @parameter frameSize size parameters of the frame to fit the text into.
  *
- * @return _CTFrame* created using the given parameters
+ * @return CTFrameRef created using the given parameters
  */
-static _CTFrame* _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
+static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
     RETURN_NULL_IF(!string);
 
-    _CTFrame* frame = [[_CTFrame new] autorelease];
-    if (range.length <= 0) {
-        return frame;
-    }
+    __CTFrame* frame = __CTFrame::CreateInstance(kCFAllocatorDefault);
+    CFAutorelease(frame);
```

Comment:
Since it's returning and auto released object, i'd rather not AutoCF it it then detach it and autorelease it.

---
In reply to: [110446169](https://github.com/Microsoft/WinObjC/pull/2427#discussion_r110446169) [](ancestors = 110446169)